mne/epochs.py

```diff
@@ -37,6 +37,7 @@
 from .channels.channels import (ContainsMixin, UpdateChannelsMixin,
                                 SetChannelsMixin, InterpolationMixin)
 from .filter import resample, detrend, FilterMixin
+from .decoding.mixin import VectorizerMixin
```

Comment:
It works after deleting the import. How do we tackle the circular import problem?